src/reducers/loadingReducer/loadingReducer.js

```diff
@@ -0,0 +1,8 @@
+export const loadingReducer = (state = true, action) => {
+  switch (action.type) {
```

Comment:
Do we want the default state here to be false to start? Then when we invoke the fetch call, we will set it to true?

Reply:
Good catch, I will update this.